src/arch/arm/config.cmake

```diff
@@ -83,7 +88,8 @@ config_option(
     KernelArmHypervisorSupport ARM_HYPERVISOR_SUPPORT
     "Build as Hypervisor. Utilise ARM virtualisation extensions to build the kernel as a hypervisor"
     DEFAULT ${default_hyp_support}
-    DEPENDS "KernelArmCortexA15 OR KernelArmCortexA35 OR KernelArmCortexA57 OR KernelArmCortexA53"
+    DEPENDS
+        "KernelArmCortexA15 OR KernelArmCortexA35 OR KernelArmCortexA57 OR KernelArmCortexA53 OR KernelArmCortexA72"
```

Comment:
Have the hypervisor functions been tested on A72 (i.e., bringing up a Linux VM)?

Reply:
So far this has not been tested in detail. We are just assuming the A72 basically works like the other ARMv8 cores. Also, internally we have some ongoing internal activity about this, but that is still in the early stages. Maybe we should keep this out of the current commit for now. Is there any quick way to test this vie seL4test?

Reply:
I was imaging someone turns on this option on RPi4 but finds out that it does not work. It should be quite similar, but you never know until you actually run it. I suggest we keep this option off until we confirm that we can run a VM on it.

Reply:
Created https://github.com/seL4/seL4/issues/299 to track this

Reply:
Created #299 to track this